lib/cli/src/cli.rs

```diff
@@ -218,6 +218,8 @@ pub fn wasmer_main() {
 }
 
 fn wasmer_main_inner() -> Result<(), anyhow::Error> {
+    crate::logging::set_up_logging();
```

Comment:
Not sure about this.

The "do not show any logs if not enabled" restriction is primarily relevant for `run`, for the others having a default level of WARN actually makes sense.

Reply:
That'll make things pretty annoying to maintain because we'll need to remember to initialize logs for every sub-command in the CLI, plus any sub-commands we may add in the future.

What are your thoughts on still running `set_up_logging()` from `wasmer_main_inner()`, but we'll make the `verbosity` flags global so anyone can do `--verbose` for any command? Instead of going through `clap-verbosity-flags`, I might add some custom logic so no `--verbose` flag leaves logs off, one occurrence means INFO logs, two means DEBUG, and so on. With this setup it probably doesn't make sense to have a `--quiet` flag.

Reply:
I’m a bit more agnostic on the result here